hell/adventcalendar/2025/13/index.md

```diff
@@ -1,35 +1,120 @@
 ---
 title: "A11y Considerations in Math on the Web"
-author: "Your Name"
-author_bio: "Your short bio"
+author: "Manuel Sánchez"
+author_bio: "Accessibility specialist and game developer at [DIE ZEIT](https://www.zeit.de/spiele/index). Currently creating [The Runic Edda](https://www.therunicedda.com), a solo game blending storytelling, Viking history, and fun. I am driven by curiosity, creativity, and the belief that technology should welcome everyone."
 date: 2025-12-13
 author_links:
-  - label: "Site"
-    url: "https://linktoyourblog123.com"
-    link_label: "linktoyourblog123.com"
-intro: "<p>Short introductory text</p>"
+  - label: "Web"
+    url: "https://www.manuelsanchezdev.com"
+    link_label: "manuelsanchezdev.com"
+  - label: "Bluesky"
+    url: "https://bsky.app/profile/manuelsanchezdev.com"
+    link_label: "@manuelsanchezdev.com"
+intro: "<p>Math on the web has always been a visual and accessibility challenge. In this article, we learn how to make structures understandable for assistive technologies by using today's MathML Core.</p>"
 image: "advent25_13"
 ---
-Some text.
-Some text.
 
-Some text. Some text.
+Maybe it has happened to you that you wanted to write some formulas in HTML to display them on a website, and even though there are multiple ways to do it, accessibility is often not considered in the process. How the formula is read by screen readers is crucial to ensure that we don't leave anyone behind.
 
-## Heading
+Even though the web is full of many different and interesting approaches, the reality is that [MathML](https://developer.mozilla.org/en-US/docs/Web/MathML) is usually the best option for this problem. It has its own syntax and was not originally designed for the web, but MathML provides various elements that give the correct semantics to the different parts of a formula.
```

Comment:
> MathML is usually the best option

Your article is about accessibility, so I have to assume that this statement also implies that it's the best option in terms of accessibility. Is this true? You've included a screenshot of the accessibility tree, but what about actual screen reader support? Have you tested this with common SR/browsers combinations?